From 9c6aec613f29f13a2d74c83d19a808d3a6213f9c Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Tue, 12 Nov 2019 23:40:56 +1300
Subject: Remove binary targets to avoid pulling in test dependencies

In future this patch won't be necessary, but as-is, dev-dependencies
are auto-required when any binary target exists.

This patch simply removes binary targets to avoid that problem.
---
 Cargo.toml  |  7 -------
 src/main.rs | 13 -------------
 2 files changed, 20 deletions(-)
 delete mode 100644 src/main.rs

diff --git a/Cargo.toml b/Cargo.toml
index ab0be71..18b4352 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,13 +33,6 @@ debug = true
 [lib]
 name = "aho_corasick"
 
-[[bin]]
-name = "aho-corasick-dot"
-path = "src/main.rs"
-test = false
-bench = false
-doc = false
-
 [[bench]]
 name = "bench"
 path = "benches/bench.rs"
-- 
2.24.0

